Resources/doc/installation.rst

```diff
@@ -78,6 +78,20 @@ the MongoDB ODM across your application:
     Of course, you'll also need to make sure that the MongoDB server is running
     in the background. For more details, see the MongoDB `Quick Start`_ guide.
 
+
+Authorization
```

Comment:
This should be [Authentication](https://docs.mongodb.com/manual/core/authentication/). "Authorization" would more correctly pertain to [role-based access control](https://docs.mongodb.com/manual/core/authorization/), which doesn't really involve the driver since roles are configured entirely on the server. This quote from the manual succinctly explains the difference between the two:

> Authentication is the process of verifying the identity of a client. When access control, i.e. authorization, is enabled, MongoDB requires all clients to authenticate themselves in order to determine their access.

Reply:
Agreed, will change in the next commit.